app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  <%= simple_form_for group do |f| %>
+    <%= f.input :name %>
+    <%= f.input :responsable_name %>
+    <%= f.input :responsable_contact %>
+    <%= f.input :from_date, order: [:day, :month, :year] %>
+    <%= f.input :to_date, order: [:day, :month, :year] %>
+    <%= f.input :notes %>
+    <%= f.submit t(".submit") %>
+    <% end %>
```

Comment:
Indentation

Reply:
Done in fixup 643c458

Reply:
`<% end %>` is still not indented correctly :/

Reply:
Oh, I was waiting to finish all the changes to push it! :grimacing:
I will do one by one, roger that! :)